man/cc-test-reporter-format-coverage.1.md

```diff
@@ -33,8 +33,7 @@ tests were run. Defaults to current working directory.
 
 ## COVERAGE_FILE
 
-Path where the coverage file is located. It requires the flag `--input-type`
-to be set.
+(Optional) Explicit path where the coverage file is located. If not passed, the path to the file is inferred based on standard some coverage tool locations. If specifying an explicit COVERAGE_FILE, an `--input-type` must also be specified.
```

Comment:
@pbrisbin by enforce you mean raise an error? currently i will ignore argument and go for the inference logic